worker.py

```diff
@@ -210,9 +210,12 @@ def main(self):
             if self.error_code and self.seen_per_cycle:
                 self.error_code = None
             self.step += 1
-            time.sleep(
-                random.uniform(config.SCAN_DELAY, config.SCAN_DELAY + 2)
-            )
+            if isinstance(config.SCAN_DELAY, tuple):
```

Comment:
I don't like this `if`. Especially in this place. I think it's better to silently convert config.SCAN_DELAY into tuple if it's an integer, somewhere below `if __name__ == '__main__':` (line 374).